core/server/middleware/index.js

```diff
@@ -70,6 +71,12 @@ setupMiddleware = function setupMiddleware(blogApp, adminApp) {
         }
     }
 
+    // Preload link headers
+    blogApp.use(netjet({
```

Comment:
I'm more than happy to update this PR to make these values configurable via `config.js` (and also, if desired, to disable  `netjet`), but I haven't exactly determined the preferred way to do so. :smile: